int32 LeftFront
int32 LeftRear
int32 RightFront
int32 RightRear
